functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
Yep, just tested it and the job schedule is pointing to the shared collection

````
PS C:\WINDOWS\system32> $server.jobserver.jobs.jobschedules |Get-Member


  TypeName: Microsoft.SqlServer.Management.Smo.Agent.JobSchedule

[1:58] 
PS C:\WINDOWS\system32> $server.jobserver.SharedSchedules | Get-Member


  TypeName: Microsoft.SqlServer.Management.Smo.Agent.JobSchedule
````